dat/events/neutral/news.lua

```diff
@@ -1,238 +1,115 @@
-                              }
-header_table["Independent"] = {_("Welcome to Universal News Feed. All the headlines all the time.")
-                              }
-header_table["Empire"] =      {_("Welcome to the Empire News Centre.")
-                              }
-header_table["Dvaered"] =     {_("Welcome to the Dvaered News Centre. All that happens. In simple words. So you can understand.")
-                              }
-header_table["Goddard"] =     {_("Welcome to Goddard News Centre. We bring you the news from around the Empire.")
-                              }
-header_table["Pirate"] =      {_("Pirate News. News that matters.")
-                              }
-header_table["Sirius"] =      {_("Sirius News Reel. Words of the Sirichana for all.")
-                              }
-header_table["FLF"] =         {_("The word of the Free Resistance.")
-                              }
-header_table["Frontier"] =    {_("News from the Frontier Alliance.")
-                              }
-header_table["Proteron"] =    {_("Word from the Proteron state.")
-                              }
-header_table["Za'lek"] =      {_("Scientific, Socioeconomic, and Sundry Events")
-                              }
-header_table["Soromid"] =     {_("The voice of the future.")
-                              }
-header_table["Thurion"] =     {_("Data Relay Network")
-                              }
-
-greet_table={}
-
-greet_table["Generic"] =      {""
-                              }
-greet_table["Independent"] =  {""
-                              }
-greet_table["Empire"] =       {_("Fresh news from around the Empire."),
-                              _("Remembering the Incident."),
-                              _("Keeping you informed.")
-                              }
-greet_table["Dvaered"] =      {""
-                              }
-greet_table["Goddard"] =      {""}
-greet_table["Pirate"] =       {_("News that matters."),
-                              _("Adopt a cat today!"),
-                              _("Laughing at the Emperor."),
-                              _("On top of the world."),
-                              _("Piracy has never been better.")
-                              }
-greet_table["FLF"] =          {""
-                              }
-greet_table["Frontier"] =     {_("News you can trust.")
-                              }
-greet_table["Sirius"] =       {_("Stay faithful."),
-                              _("Sirichana watches and guides you.")
-                              }
-greet_table["Proteron"] =     {""
-                              }
-greet_table["Za'lek"] =       {""
-                              }
-greet_table["Soromid"] =      {_("Genetically tailoured for you.")
-                              }
-greet_table["Thurion"] =      {""
-                              }
-
-articles={}
+articles = {}
 
 articles["Generic"] = {
    --[[
       Science and technology
    --]]
    {
+      tag = "Techs Probe for Sol",
```

Comment:
I have added gettext.gettext_noop which is basically a way to get gettext to detect the sentence for translation, but doesn't translate it. This should let you do something like
```
tag = gettext.gettext_noop("Tech Probe for Sol")
title = _(tag)
desc = _("Technicians...")
```

Should be possible to just make the titles be untranslated, and translate it when adding the news. Or even translating it C-side.

Reply:
Ah, yeah, that should be useful!

Actually there might be some other improvements we could make with that... hold that thought, we'll look into that today.

🕵️

Reply:
Alright, I've added in a use of something like this. Not in the way you described because that way's actually impossible, but by using tag to generate the title if title is unset.

🕵️